### ISSUE TITLE
crazy idea: dataverse-metrics should pull from authoritative installation list

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse-metrics#76]
https://github.com/IQSS/dataverse-installations/blob/master/data/data.json is more complete and current.

Note that dataverse-installations' map checks for an installation in the dataverse-metrics JSON, and if found, includes a link to dataverse.org/metrics in that installation's display.

**Comment by PaulBoon** [comment]
There is this script that updates; https://github.com/IQSS/dataverse-metrics/blob/master/global/update-all-installations-list.sh. But this must be run from time to time, seems that https://metrics.dataverse.org has not done that. Also we should update with every release, maybe soon. 
The other approach is to always have this empty and forcing you to run that script it al least once when installing.

**Comment by donsizemore** [comment]
@PaulBoon not crazy at all. IIRC things weren't set up this way because at the time not all installations were new enough (4.11+?) to support metrics. I support this change.

**Comment by donsizemore** [comment]
@PaulBoon there was a host of operational issues. I do like the idea of an option to tell dataverse-metrics to get its list from the canonical source, though.

**Comment by PaulBoon** [comment]
@donsizemore Ahh, you mean that the python code could do the processing every time it's run (crontab probably). 
About the field that might be added, I meant to be added to that data.json so we can automate more precisely, not sure how that was produced though.

**Comment by donsizemore** [comment]
@PaulBoon it was a crazy idea. On Friday I diffed the hostnames from metrics' current config from the all-installations JSON, and wound up removing about half of what I added. Installations aren't running 4.11+ to offer the metrics URL, or they block HTTP calls to the metrics API endpoints, and a couple appear to be running a fork and/or returning non-standard data. So... the list is more current than it was. Once the scripts run cleanly I'll copy that list over into the sample config.
